CHANGELOG.rst

```diff
@@ -8,6 +8,8 @@ Updated
 ~~~~~~~
 
 - Remove unused `scipy` dependency
+- Break out Jupyter objects into individual files. This affects direct imports from
```

Comment:
So this will be a new major version? Seems like a breaking change.

Reply:
As mentioned in the commit message, the usual imports are not broken. That said, any direct imports will break, so we should denote this but this is a major change